src/components/header/bottom-header/left-menu/left-menu.tsx

```diff
@@ -1,22 +1,61 @@
-import styles from '../../header.module.css';
+import styles from './left-menu.module.css';
 import { NavLink } from 'react-router-dom';
 import { leftLinks } from './left-links-list';
 import classNames from 'classnames';
+import { SvgBuilder } from '../../../svg-builder/svg-builder';
+import { IconType } from '../../../svg-builder/enums';
+import { useState } from 'react';
+import { Button } from '../../../button/button';
+import { useEffect } from 'react';
 
 export const LeftNavMenu = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  useEffect(() => {
+    const handleClickOutside = (event: Event) => {
+      if (
+        event.target instanceof Element &&
+        !event.target.closest(`.${styles.navigationLeftMenu}`) &&
+        !event.target.closest(`.${styles.burgerMenu}`)
+      ) {
+        setIsMenuOpen(false);
+      }
+    };
```

Comment:
```suggestion
        !event.target.closest(`.${styles.burgerMenu}`)
```
тут 3 момента
1 - это условие не нужно. как будто если клик по бургеру закроет меню - это ожидаемое поведение
2 - в принипе странно, что бургер над оверлеем и на него можно кликнуть, при твоем дизайне
3 - весь этот юз эффект ничего не делает. Эта конструкция имеет смысл, если оверлея нет. тогда вешаем слушатель на весь документ и проверяем куда мы там кликнули, чтоб закрыть меню. но раз ты все, кроме самого меню, закрываешь оверлеем, то мы никуда кроме него и кликнуть не можем. а у тебя на нем и так висит закрытие менюхи. можешь его закомментить (весь useEffect) и посмотреть изменится ли что-нибудь